guide/07-working-with-bigdata/10-extending_geoanalytics_with_custom_tools.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>In the example below, a Living Atlas layer containing congressional district boundaries is converted to a DataFrame using its URL. <code>pyspark</code> is then used to calculate statistics on two columns in the dataset. These statistics are then used to query the dataset to find districts of interest before writing out the result to a big data file share.</p><p> </p><p><strong>Querying a layer using pyspark.sql</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>pyspark</p>